src/pages/Home.jsx

```
@@ -1 +1,6 @@
-// 홈 화면 입니다.
\ No newline at end of file
+// 홈 화면 입니다.
+import styled from "styled-components";
```

ESLint

[error] 'styled' is defined but never used. Allowed unused vars must match /^[A-Z_]/u.

(no-unused-vars)